utils/lib/Quickstart.ts

```diff
@@ -42,36 +42,57 @@ const SUPPORT_LEVEL_ENUMS: SupportLevelMap = {
 type ComponentType = typeof Alert | typeof Dashboard | typeof DataSource;
 type Components = InstanceType<ComponentType>;
 
+enum ConfigKey {
+  DataSource = 'dataSourceIds',
+  AlertPolicies = 'alertPolicies',
+  Dashboards = 'dashboards', 
+};
+
 interface ConfigToMutationMap {
-  configKey: string;
+  configKey: ConfigKey;
   mutationKey: string;
   ctor: ComponentType;
-}
+};
 
 const ConfigToMutation: ConfigToMutationMap[] = [
-  { configKey: 'alertPolicies', mutationKey: 'alertConditions', ctor: Alert },
-  { configKey: 'dashboards', mutationKey: 'dashboards', ctor: Dashboard },
   {
-    configKey: 'dataSourceIds',
+    configKey: ConfigKey.AlertPolicies,
+    mutationKey: 'alertConditions',
+    ctor: Alert,
```

Comment:
This is just the constructor, right? Could we just rename it to `constructor`, `ctor` feels needlessly cryptic

Reply:
`ctor` is what we had originally, i think there was a reason for this but im happy to address it in this PR

Reply:
I guess my commit messaged left the chat

https://github.com/newrelic/newrelic-quickstarts/pull/1649/commits/549921f036735f483af7de61afe7de040101fce2